arangod/RestHandler/RestUsersHandler.cpp

```diff
@@ -495,19 +523,44 @@ RestStatus RestUsersHandler::deleteRequest(auth::UserManager* um) {
       // revoke a user's permissions
       std::string const& db = suffixes[2];
       std::string coll = suffixes.size() == 4 ? suffixes[3] : "";
+
       if (!isAdminUser()) {
         generateError(rest::ResponseCode::FORBIDDEN, TRI_ERROR_FORBIDDEN);
+
         return RestStatus::DONE;
       }
 
+      // validate that the collection is present
+      if (suffixes.size() > 3) {
+        auto* databaseFeature = arangodb::application_features::ApplicationServer::lookupFeature<
+          arangodb::DatabaseFeature
+        >("Database");
```

Comment:
Why don't you just use DatabaseFeature::DATABASE

Reply:
I want to explicitly avoid doing that since public static variables are always a pain for reuse during testing